src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/CatalogPriceRule/Blocks/form.html.twig

```diff
@@ -0,0 +1,98 @@
+{#**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+{% import '@PrestaShop/Admin/macros.html.twig' as ps %}
+
+{{ form_start(catalogPriceRuleForm) }}
+<div class="card">
+  <h3 class="card-header">
+    <i class="material-icons">attach_money</i>
+    {{ 'Catalog price rules'|trans({}, 'Admin.Catalog.Feature') }}
```

Comment:
I guess that I'm asking the impossible but there is a lot of feedback about the wording in this section of the back office. Difference is not easy to tell between 'cart rules' (= promo codes) and 'catalog price rules' (= reductions), could the wording be changed there?

I know that those two wordings are referenced in many other places in the back office so it would be complicated but I just wanted to ask. :-)

Reply:
@matks, wdyt?

Reply:
It seems pretty ambitious to change the whole wording, but perhaps we could improve it. It is not live now but empty states will help users to make the difference, I've just checked. So it will be kind of fixed in future versions. But let's keep the remark in mind for the next major version!

Reply:
@LouiseBonnard I have one more suggestion (not associated with this PR, but with renaming), maybe 'credit slips' should also be considered to be renamed to `refunds`.

Reply:
@zuk3975, thanks, Julius, I'll take good note of it in case renaming will be possible in future versions. :-)